### DISCUSSION TITLE
Licensing for Enterprise Edition is unclear

### DISCUSSION THREAD

**Discussion by ccharles** [dbeaver/dbeaver#390] [Q&A] [answered]
The [download page](http://dbeaver.jkiss.org/download/) says:

> You may choose between Community and Enterprise editions. Both versions are free. Enterprise Edition includes NoSQL databases support.
> Also you can use GitHub mirror.

and then:

> Community Edition sources, issue tracker, etc: https://github.com/serge-rider/dbeaver

Where are the Enterprise Edition sources? And are they licensed under the GPL v2 [like the Community Edition is](https://github.com/serge-rider/dbeaver/blob/devel/LICENSE.md)?

Finally, is there a reason that `.rpm` and `.deb` packages are provided for the Community Edition but only `.tar.gz` for the Enterprise Edition? Do you need help with packaging, or is this related to licensing?

**Comment by serge-rider** [comment]
CE version is GPL. And always will be.
EE version is not open-source, so there is no any open-source license for it. For now EE version is free, i.e. you don't have to pay for it. Things may change in a future but we don't have any particular short term plans.

I'm going to make these questions more clear shortly (web site will be updated).

In regards to EE packaging. There is only one (and pretty silly) reason why there are no deb/rpm packages - disk space occupation and build time. We are using very cheap web hosting (poor we :) ).
But there are plans to switch to another hosting and/or move most of the stuff to GitHub.

**Comment by ccharles** [comment]
Thank you for the update @serge-rider. I'm glad to hear that you are planning to make the website more clear about this.

I notice that you added the "help wanted" tag to this issue. What would be the best way to help?

**Comment by serge-rider** [comment]
"Help wanted" is the legacy name from the standard github labels, just changed it to the "question" to avoid misunderstanding.

**Comment by bsed** [comment]
how i pay ee license

**Comment by serge-rider** [comment, marked as answer]
@bsed you can do it on official web site https://dbeaver.com/purchase/
Also check community info in https://github.com/serge-rider/dbeaver/wiki/Enterprise-Edition

**Comment by bsed** [comment]
@serge-rider  oh, thanks
